portability-spi-transfer/src/main/java/org/datatransferproject/spi/transfer/types/signals/SignalType.java

```diff
@@ -0,0 +1,7 @@
+package org.datatransferproject.spi.transfer.types.signals;
+
+public enum SignalType {
```

Comment:
(see next comments for context) here's a way we could maintain one chunk of unified logic insdie DTP that delivers both views of a job's lifecycle.

```
public enum JobLifecycleChange {
   ...._PERFECT,
  ... _PARTIAL,
  ..._SKIPPED,

}
```


```java
/** Whether the job is running still. All other nuance should be captured elsewhere. */
public enum JobLifecycle {
  JOB_STARTED,
  JOB_DONE
}

public static JobLifecycle from(JobLifecycleChange change) {
}
```